tools/src/test/resources/dev/run-qualification-tests.sh

```diff
@@ -0,0 +1,395 @@
+RELATIVE_PROF_LOG_PATH="src/test/resources/spark-events-profiling"
+RELATIVE_QUAL_REF_PATH="src/test/resources/QualificationExpectations"
+RELATIVE_TOOLS_JAR_PATH="tools/target/spark311/"
+RELATIVE_COMMON_JAR_PATH="common/target/spark330/"
+
+
+PROJECT_ROOT=""
+MODULE_PATH=""
+PROF_LOG_DIR=""
+QUAL_LOG_DIR=""
+QUAL_REF_DIR=""
+JVM_DEFAULT_HEAP_SIZE="10g"
+
+# get the directory of the script
+WORK_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]:-$0}"; )" &> /dev/null && pwd 2> /dev/null; )";
+BATCH_OUT_DIR="${WORK_DIR}/qualification-output"
+CSV_OUT_DIR="${BATCH_OUT_DIR}/csv"
+RUNS_OUT_DIR="${BATCH_OUT_DIR}/runs-output"
+
+java_cp=""
+rapids_jar_home=""
+script_out_dir=""
+qual_events_dir=""
+prof_events_dir=""
+qual_expectations_dir=""
+
+heap_size=$JVM_DEFAULT_HEAP_SIZE
+declare -A qualification_path_map
+
+show_help()
+{
+   # Display Help
+   log_msg
+   log_msg "Run the Qualification tool to generate CSV files to update/replace Qualification expectations set."
+   log_msg "The output folder of the CSV files is test/resources/dev/qualification-output/csv"
+   log_msg
+   log_msg "The script compares the CSV files. If pairs of files do not match or the directories are"
+   log_msg "not identical, the script will fail."
+   log_msg "Please make sure to update the CSV files for the failing unit tests."
+   log_msg
+   log_msg
+   log_msg "Optional Arguments:"
+   log_msg "  --rapids-jars-dir=<arg>       - Directory containing RAPIDS jars. By default the script sets it to the"
+   log_msg "                                  target directory of tools."
+   log_msg "  --cp=<arg>                    - Classpath required as dependencies to run the qualification tool."
+   log_msg "                                  For example, if \$SPARK_HOME/jars are not in the directory <rapids-jars>, then"
+   log_msg "                                  pass it as --cp=\$SPARK_HOME/jars/*:\$CLASS_PATH"
+   log_msg "  --out-dir=<arg>               - Output directory passed to the qualification tool runs."
+   log_msg "                                  By default is is set to test/resources/dev/qualification-output"
+   log_msg "                                  The directory has two subdirectories: csv/ which has the CSV files; and"
+   log_msg "                                  runs-output/ which has the actual output of the tool."
+   log_msg "                                  target directory of tools."
+   log_msg "  --qual-expectations-dir=<arg> - Path of the reference CSV files."
+   log_msg "                                  By default is is set to test/resources/QualificationExpectations"
+   log_msg "  --qual-events-dir=<arg>       - Path of the Spark events logs used as input for the unit tests"
+   log_msg "                                  By default is is set to test/resources/spark-events-qualification"
+   log_msg "  --prof-log-dir=<arg>          - Some the unit tests reads files located in another directory."
```

Comment:
nit grammar "Some the".    What happens if we just call this twice instead of having a separate option, once on qualification dir and once on files in profiling dir?  We don't use all the profiling once I thought so I assume user picks and chooses which ones to copy?

Reply:
I got rid of the second directory. Basically, if an input directory is passed to the script, then we should expect that it contains all the required log files.
I also updated the description to emphasize that.

Reply:
need to remove the option then. sorry if I was confusing, what does that mean then for this script?  we don't have all our eventlogs in one directory, so this script won't work with it?

Reply:
Or you mean by default it works with our test setup, but if someone was using it for something else, they must all be in the same directory?

Reply:
> Or you mean by default it works with our test setup, but if someone was using it for something else, they must all be in the same directory?

Yes, exactly. this is what I meant.
By default, it will work fine for our test setup loading the log files from two directories.